internal/dwarf/frame/table.go

```diff
@@ -321,7 +369,7 @@ func setloc(frame *FrameContext) {
 	var loc uint64
 	_ = binary.Read(frame.buf, frame.order, &loc)
 
-	frame.loc = loc + frame.cie.staticBase
+	frame.loc = loc
```

Comment:
I think this change requires a comment. We might be missing to pass the correct base to the parser. This changes the initial ergonomics of the package.

Reply:
The `staticBase` is passed to the parser correctly. What did go wrong here?

Reply:
My bad, left some testing code here